www/front_src/src/App.js

```diff
@@ -28,11 +32,35 @@ class App extends Component {
     this.setState({ isFullscreenEnabled: true });
   }
 
+  UNSAFE_componentWillMount = () => {
+    axios("internal.php?object=centreon_acl_webservice&action=getCurrentAcl")
+      .get()
+      .then(({data}) => this.setState({acls: data, aclsLoaded: true}))
+  }
+
+  linkRoutesAndComponents = () => {
+    const {acls} = this.state;
+    return routes.map(({ path, comp, ...rest }) => (
+      <ClassicRoute
+        history={history}
+        path={path}
+        component={acls.indexOf(`/${path.split('/_CENTREON_PATH_PLACEHOLDER_/')[1]}`) > -1 ? NotAllowedPage : comp}
```

Comment:
you could use includes ;)
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/includes

Reply:
Old school :) I think that I saw somewhere that this is the faster :) I can change if you want @kduret ? :)